test/tidelinedata_test.js

```diff
@@ -30,18 +30,21 @@ var crossfilter = require('crossfilter');
 var moment = require('moment-timezone');
 
 var types = require('../dev/testpage/types');
-var { MGDL_UNITS, MMOLL_UNITS } = require('../js/data/util/constants');
+var { MGDL_UNITS, MMOLL_UNITS, DEFAULT_BG_BOUNDS, BG_CLAMP_THRESHOLD } = require('../js/data/util/constants');
 
 var TidelineData = require('../js/tidelinedata');
 
+var bgUnits;
```

Comment:
No need to declare this variable in the `global` scope.

Can simply change line 41 to:
```
var bgUnits = MGDL_UNITS;
```